test/e2e/proxy.go

```diff
@@ -242,11 +259,9 @@ func proxyContext(version string) {
 						recordError(fmt.Sprintf("%v: path %v took %v > %v", i, path, d, proxyHTTPCallTimeout))
 					}
 				}(i, path, val)
-				// default QPS is 5
-				time.Sleep(200 * time.Millisecond)
```

Comment:
maybe default QPS went way up since this test was last touched... not sure how to check that in the test to make sure we don't flake/fail on client side QPS